src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.model.Movie;
+import mate.academy.util.exceptions.DataProcessingException;
+
+public interface MovieDao {
+    Movie add(Movie movie) throws DataProcessingException;
```

Comment:
```suggestion
    Movie add(Movie movie);
```
do we need this here?